bastionPolicy.tf

```diff
@@ -1,5 +1,9 @@
-data "aws_iam_group" "AllowTakeonBastionAccess" {
-    group_name = "AllowTakeonBastionAccess"
+# data "aws_iam_group" "AllowTakeonBastionAccess" {
+#     group_name = "AllowTakeonBastionAccess"
```

Comment:
Can this commented code be removed

Reply:
Yep, no idea why I still had it left in! I shall update this now